projects/approved-premises-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/integrations/delius/person/ProbationCase.kt

```diff
@@ -115,15 +115,34 @@ class CommunityManagerTeam(
 
     val description: String,
 
+    @ManyToOne
+    @JoinColumn(name = "district_id")
+    val ldu: Ldu,
+
     @Id
     @Column(name = "team_id")
     val id: Long
 )
 
+@Immutable
+@Entity
+@Table(name = "district")
+class Ldu(
```

Comment:
Is LDU the right name for this?  I thought it was called "Local Admin Unit" now.

Reply:
No worries, I think actually Consider a Recall refer to it as LDU too - so I guess we can treat LDU/LAU as interchangeable.